src/test/java/org/broadinstitute/hellbender/tools/walkers/sv/SVStratifyIntegrationTest.java

```diff
@@ -0,0 +1,208 @@
+import org.broadinstitute.hellbender.testutils.VariantContextTestUtils;
+import org.broadinstitute.hellbender.tools.copynumber.arguments.CopyNumberStandardArgument;
+import org.broadinstitute.hellbender.tools.spark.sv.utils.GATKSVVCFConstants;
+import org.broadinstitute.hellbender.tools.sv.stratify.SVStratificationEngineArgumentsCollection;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import picard.vcf.VcfUtils;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class SVStratifyIntegrationTest extends CommandLineProgramTest {
+
+    @Test
+    public void testBwaMeltCohort() {
+        final File outputDir = createTempDir("stratify");
+        final String inputVcfPath = getToolTestDataDir() + "bwa_melt.chr22.vcf.gz";
+        final String configFile = getToolTestDataDir() + "test_config.tsv";
+
+        final String segdupFile = getToolTestDataDir() + "hg38.SegDup.chr22.bed";
+        final String segdupName = "SD";
+        final String repeatmaskerFile = getToolTestDataDir() + "hg38.RM.chr22_subsampled.bed";
+        final String repeatmaskerName = "RM";
+
+        final ArgumentsBuilder args = new ArgumentsBuilder()
+                .addOutput(outputDir)
+                .add(CopyNumberStandardArgument.OUTPUT_PREFIX_LONG_NAME, "test")
+                .add(SVStratify.SPLIT_OUTPUT_LONG_NAME, true)
+                .add(SVStratificationEngineArgumentsCollection.STRATIFY_CONFIG_FILE_LONG_NAME, configFile)
+                .add(SVStratificationEngineArgumentsCollection.CONTEXT_NAME_FILE_LONG_NAME, segdupName)
+                .add(SVStratificationEngineArgumentsCollection.CONTEXT_INTERVAL_FILE_LONG_NAME, segdupFile)
+                .add(SVStratificationEngineArgumentsCollection.CONTEXT_NAME_FILE_LONG_NAME, repeatmaskerName)
+                .add(SVStratificationEngineArgumentsCollection.CONTEXT_INTERVAL_FILE_LONG_NAME, repeatmaskerFile)
+                .add(SVStratificationEngineArgumentsCollection.OVERLAP_FRACTION_LONG_NAME, 0.5)
+                .add(StandardArgumentDefinitions.SEQUENCE_DICTIONARY_NAME, GATKBaseTest.FULL_HG38_DICT)
+                .add(StandardArgumentDefinitions.VARIANT_LONG_NAME, inputVcfPath);
+
+        runCommandLine(args, SVStratify.class.getSimpleName());
+
+        final File[] outputFiles = outputDir.listFiles();
+        Assert.assertEquals(outputFiles.length, 14);
+        final Map<String, Integer> expectedOutputSuffixes = new HashMap<>();
+        expectedOutputSuffixes.put("INS_small_SD", 46);
+        expectedOutputSuffixes.put("DEL_50_5k_both", 110);
+        expectedOutputSuffixes.put("DEL_5k_50k_SD", 2);
+        expectedOutputSuffixes.put("DUP_lt5kb_RM", 0);
+        expectedOutputSuffixes.put("INV_gt1kb", 26);
+        expectedOutputSuffixes.put("BND_SD", 77);
+        expectedOutputSuffixes.put(SVStratify.DEFAULT_STRATUM, 1196);
```

Comment:
I'm going to trust that you verified this with an orthogonal python script or somesuch

Reply:
Yeah I've been testing against the gatk-sv pipeline results, although those evaluations are qualitative. I trust this will alert us if any unintentional changes happen though.